lib/puppet/provider/package/pip3.rb

```diff
@@ -0,0 +1,22 @@
+# Puppet package provider for Python's `pip3` package management frontend.
+# <http://pip.openplans.org/>
+
+require 'puppet/provider/package/pip'
+
+Puppet::Type.type(:package).provide :pip3,
+  :parent => :pip do
+
+  desc "Python packages via `pip3`.
+
+  This provider supports the `install_options` attribute, which allows command-line flags to be passed to pip3.
+  These options should be specified as a string (e.g. '--flag'), a hash (e.g. {'--flag' => 'value'}),
+  or an array where each element is either a string or a hash."
+
+  def self.cmd
+    if Facter.value(:osfamily) == "RedHat" and Facter.value(:operatingsystemmajrelease).to_i < 7
+      "pip-python3"
```

Comment:
For clarification, I was thinking we should be checking if `pip-python3` is available, if not, check if `pip3` is available, rather than just looking for `pip-python3` for el < 7